specificatie/nationaliteit.yaml

```diff
@@ -0,0 +1,80 @@
+      discriminator:
+        propertyName: type
+        mapping:
+          GbaNationaliteit: '#/components/schemas/GbaNationaliteit'
+          Nationaliteit: '#/components/schemas/Nationaliteit'
+          BehandeldAlsNederlander: '#/components/schemas/BehandeldAlsNederlander'
+          VastgesteldNietNederlander: '#/components/schemas/VastgesteldNietNederlander'
+    Nationaliteit:
+      description: |
+        * **redenOpname** : De reden op grond waarvan de persoon de nationaliteit gekregen heeft.
+      allOf:
+        - $ref: '#/components/schemas/AbstractNationaliteit'
+        - type: object
+          properties:
+            datumIngangGeldigheid:
+              $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+            nationaliteit:
+              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+            inOnderzoek:
+              $ref: '#/components/schemas/NationaliteitInOnderzoek'
+    BehandeldAlsNederlander:
+      allOf:
+        - $ref: '#/components/schemas/AbstractNationaliteit'
+        - type: object
+          properties:
+            datumIngangGeldigheid:
+              $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+    VastgesteldNietNederlander:
+      allOf:
+        - $ref: '#/components/schemas/AbstractNationaliteit'
+        - type: object
+          properties:
+            datumIngangGeldigheid:
+              $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+    GbaNationaliteit:
+      description: |
+        Bij de LO GBA versie kunnen datums, bijv datumIngangGeldigheid de LoGbaDatum refereren
+        Het hoeft niet van AbstractDatum af te leiden omdat het nooit een andere type kan zijn
+      allOf:
+        - $ref: '#/components/schemas/AbstractNationaliteit'
+        - type: object
+          properties:
+            datumIngangGeldigheid:
+              $ref: 'datum.yaml#/components/schemas/GbaDatumBasis'
+            nationaliteit:
+              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+            aanduidingBijzonderNederlandschap:
+              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+            inOnderzoek:
+                $ref: 'gba-inonderzoek.yaml#/components/schemas/GbaInOnderzoek'
+    NationaliteitInOnderzoek:
+      type: object
+      description: |
+        Geeft aan welke gegevens over de nationaliteit in onderzoek zijn. Zie de [functionele specificaties](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.1.0/features/in_onderzoek.feature)
+      properties:
+        aanduidingBijzonderNederlanderschap:
```

Comment:
bij het type Nationaliteit wordt nu de mogelijkheid geboden dat aanduidingBijzonderNederlanderschap in onderzoek staat. Dat lijkt mij zeer onwaarschijnlijk, behalve natuurlijk wanneer de hele categorie Nationaliteit in onderzoek staat en daarmee impliciet ook aanduidingBijzonderNederlanderschap. 
Ik zou niet verwachten dat het voor kan komen dat een nationaliteit gevuld is en daarbij alleen element aanduidingBijzonderNederlanderschap in onderzoek staat. Nationaliteit en aanduidingBijzonderNederlanderschap kunnen immers niet tegelijkertijd voorkomen (in één categoriestapel), dus wanneer het onderzoek tot conclusie heeft dat er sprake is van aanduidingBijzonderNederlanderschap en niet van nationaliteit, zou de nationaliteit beëindigd worden.
@CathyDingemanse durven we het risico te nemen in de specificaties dat nooit iemand op het geniale idee is gekomen element aanduidingBijzonderNederlanderschap in onderzoek te zetten wanneer nationaliteit gevuld is?

Reply:
Ja, als jij bij RvIG navraagt of dit in de database voorkomt.